info.femm.unofficial-manifest.yml

```diff
@@ -0,0 +1,235 @@
+app-id: info.femm.unofficial
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: femm.sh
+build-options:
+  env:
+    - "WINEPREFIX": "/var/data/.local/share/femm42-flatpak"
+    - "WINEDLLOVERRIDES": "mscoree,mshtml="
+    - "V": "1"
+
+#rename-desktop-file: wine.desktop
+
+# Support 32-bit at runtime
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
```

Comment:
you do you need this?

Reply:
I ask again: why do you need this (conversation was resolved wihtout resolution or answer)